src/metadataparsers/mods/Mods.php

```diff
@@ -151,24 +171,15 @@ public function determineRepeatedWrapperChildElements($xml, $uniqueChildNodeSign
      */
     public function consolidateWrapperElements($wrapperElementArray)
     {
-        $consolidatedWrapperElementsArray = array();
+
         $elementNameTrackingArray = array();
         $elementSignatureTrackingArray = array();
         foreach ($wrapperElementArray as $wrapperElement) {
-            $name = $wrapperElement->nodeName;
-            $attributesNodeMap = $wrapperElement->attributes;
-            $length = $attributesNodeMap->length;
-            for($i=0; $i < $length; ++$i){
-               $attributeItem = $attributesNodeMap->item($i);
-               $attributeName = $attributeItem->name;
-               $attributeValue = $attributeItem->value;
-            }
-
```

Comment:
This code block seems to have been duplicated, so I kept the second instance (below) and removed this.